src/error_detectors/prod_error_detector.ts

```diff
@@ -88,23 +88,21 @@ export class ProdErrorDetector extends ErrorDetector {
         });
       });
     }
```

Comment:
I moved the reverse functionality to the other function because I noticed that in some particular cases strictNullChecks, sometimes the error can appear on too many possible node types for me to take account of, so I had to filter out nodes by length instead of kind in those particular cases. However, the reverse functionality was needed whether or not I filtered out by kind or not.